app/controllers/MovementsController.scala

```diff
@@ -68,6 +72,25 @@ class MovementsController @Inject()(
       case _                                                           => false
     }
 
+  private def SubmissionResultHandler(
```

Comment:
typically defs start lowercase

Reply:
Either lowercase this, or if we want to be extra clever, check the controller tests to see if they track the auditing - if they don't, lets move this into its own class that can be instantiated as this is used, but has its own suite of unit tests to check the audits, etc.

Reply:
I'd avoid moving this to a separate class - it's very much the concern of the controller to decide what to return.  Are we also 100% confident the different methods will never want to handle the result differently in the future?  DRY's all well and good, but you're also coupling the behaviour of 2 distinct methods - just worth a sanity check to make sure that's valid!  If we're happy they really ought to be the same, then crack on! (with the lowercasing Rob suggests - and maybe using a more active name, e.g. handleSubmissionResult?)